.github/workflows/webpage.yml

```diff
@@ -9,13 +9,16 @@ on:
   - cron: "0 21 * * *"
   push:
     branches:
-    - master
+    - reactjs
 
 jobs:
   webpage:
-    if: github.repository == 'aiidateam/aiida-registry'
+    if: github.repository == 'ahmedbasem20/aiida-registry'
```

Comment:
I guess this no longer makes sense after you merge here.

Since we already have a github page for the aiida registry repo, unfortunately we cannot just have the previous and the react version side by side https://github.com/orgs/community/discussions/47867

Maybe we can use something like this in the meanwhile? https://github.com/rossjrw/pr-preview-action